taxi/models.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"{self.name} {self.country}"

```

Comment:
Avoid overusing f-strings. The `__str__` method for the `Manufacturer` class can return `self.name` directly since it's already a string. It's more efficient and readable.